tests/py/dynamo/conversion/harness.py

```diff
@@ -60,7 +60,7 @@ def run_test(
 
             mod.eval()
             start = time.perf_counter()
-            interpreter_result = interpreter.run(precision=precision)
```

Comment:
Due to this, is `precision` in the function definition still needed?

Reply:
I guess it is not - thanks for the catch, will fix this